doc/freeband.xml

```diff
@@ -190,3 +190,35 @@ gap> ContentOfFreeBandElementCollection([x, y]);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="EqualInFreeBand">
+  <ManSection>
+    <Oper Name="EqualInFreeBand"  Arg="u, v"/>
+    <Description>
+      This operation takes a pair <A>u</A> and <A>v</A> of lists of
+      positive integers or strings, representing words in a free band.<P/>
+
+      Where <C>F</C> is a free band over some alphabet containing the
+      contents of <A>u</A> and <A>v</A>, this operation returns <E>true</E>
+      if <A>u</A> and <A>v</A> are equal in <C>F</C>, and <E>false</E>
+      otherwise.<P/>
+
+      Note that this operation is for lists and strings, as opposed to
+      FreeBandElement objects.<P/>
+
+      This is an implementation of an algorithm described by Jakub Radoszewski
```

Comment:
Can we have a proper reference here? This can be done by doing `<Cite Key="Something"/>` and by adding the appropriate XML entry to the bibliography. @MTWhyte should know how to do this, since he wrote the XML file originally.